File: app/src/main/java/cm/aptoide/pt/app/ThemeAttributeProvider.java
package cm.aptoide.pt.app;

import android.content.res.Resources;
import android.util.TypedValue;

public class ThemeAttributeProvider {

  private Resources.Theme theme;

  public ThemeAttributeProvider(Resources.Theme theme) {
    this.theme = theme;
  }

  public TypedValue getAttributeForTheme(int attributeResourceId) {
    TypedValue value = new TypedValue();
    theme.resolveAttribute(attributeResourceId, value, true);
    return value;
  }
}

File: app/src/main/java/cm/aptoide/pt/ThemeAttributeProvider.kt
package cm.aptoide.pt

import android.content.res.Resources.Theme
import android.util.TypedValue

class ThemeAttributeProvider(private val theme: Theme) {

  fun getAttributeForTheme(attributeResourceId: Int): TypedValue {
    val value = TypedValue()
    theme.resolveAttribute(attributeResourceId, value, true)
    return value
  }

}